src/main/java/cz/cvut/kbss/termit/service/business/VocabularyService.java

```diff
@@ -361,16 +362,22 @@ public List<AbstractChangeRecord> getDetailedHistoryOfContent(Vocabulary vocabul
               group = "T(ThrottleGroupProvider).getTextAnalysisVocabularyAllTerms(#vocabulary.getUri())",
               name = "allTermsVocabularyAnalysis")
     @PreAuthorize("@vocabularyAuthorizationService.canModify(#vocabulary)")
```

Comment:
Need to update the reference to the parameter here as well, otherwise it leads to NPX in the vocabulary authorization service.